src/FixturesKernel.php

```diff
@@ -61,7 +62,7 @@ private function prepareSession(Request $request): void
         $cookies = $request->cookies;
 
         if ($cookies->has($session->getName())) {
-            $session->setId($cookies->get($session->getName()));
+            $session->setId((string) $cookies->getString($session->getName()));
```

Comment:
How is this solving the `Error: Call to undefined method Symfony\Component\HttpFoundation\InputBag::getString()` fatal error for the MinkBrowserKitDriver? 

You're still using the `getString` method, which isn't available on the Symfony version installed on PHP 7.2.

Reply:
Fixed now.